### PR TITLE
fix: cargo upgrade to cargo update

### PR DESCRIPTION
**Pull request by milos018**

### CODE REVIEW

**Comment by milos018** [comment]
When running "cargo upgrade" on Ubuntu, I get "no such subcommand", did you mean "update". I then get the message "could not find "Cargo.toml" in [dir], but I guess this is expected.

File: docs/guides/getting-started/prerequisites.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Rust is installed now. Great!
 Tauri and its components can be manually updated by editing the `Cargo.toml` file or running the `cargo upgrade` command that is part of the [`cargo-edit`] tool. Open a terminal and enter the following command:
```
Comment (lucasfernog) [inline docs/guides/getting-started/prerequisites.md]:
```suggestion
Tauri and its components can be manually updated by editing the `Cargo.toml` file or running the `cargo update` command that is part of the [`cargo-edit`] tool. Open a terminal and enter the following command:
```

**Review by lucasfernog — APPROVED** [review]

**Comment by lucasfernog** [comment]
Nice catch, thank you!

**Comment by FabianLars** [comment]
wait it's not correct, we advertise the cargo-edit tool, not the built-in cargo command

**Comment by lucasfernog** [comment]
> wait it's not correct, we advertise the cargo-edit tool, not the built-in cargo command

Ahh I thought that was in another block lol I just woke up and thought i read it comes in Cargo itself :D

**Comment by FabianLars** [comment]
Yeah it's here too https://tauri.app/v1/guides/development/updating-dependencies#update-cargo-packages
idk how we ended up duping that

**Comment by lucasfernog** [comment]
I'll close this since the command is actually right, but you need to install cargo-edit first. Thanks for the PR though!

**Comment by milos018** [comment]
Glad I helped (if I helped). At least I helped you wake up :)
Great project, keep it up guys!

**Pull request closed without merging**